docker-compose.yml

```diff
@@ -67,6 +70,54 @@ services:
       - TOKEN_VERIFICATION_API_URL=http://hmpps-auth:8080/auth
       - TOKEN_VERIFICATION_ENABLED=false
       - INGRESS_URL=http://localhost:3000
+      - SERVICE_CATALOGUE_URL=http://localhost:1337
+
+
+
+
+  hmpps-service-catalogue:
+    image: quay.io/hmpps/hmpps-service-catalogue:latest
+    networks:
+      - hmpps
+    container_name: hmpps-service-catalogue
+    ports:
+      - "1337:1337"
+    env_file: .env.dpscatalogue.example
+    environment:
+      DATABASE_CLIENT: postgres
+      DATABASE_HOST: strapiDB
+      DATABASE_PORT: 5432
+      DATABASE_NAME: strapi
+      DATABASE_USERNAME: postgres
+      DATABASE_PASSWORD: password
+      JWT_SECRET: randomSecret
+      ADMIN_JWT_SECRET: randomJwtSecret
+      APP_KEYS: randomApiKey
+      NODE_ENV: development
+    depends_on:
+      - strapiDB
+
+  strapiDB:
+    container_name: strapiDB
+    platform: linux/amd64 #for platform error on Apple M1 chips
+    restart: unless-stopped
+    env_file: 
+      - .env.dpscatalogue.example
+    image: postgres:12.0-alpine
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
+      POSTGRES_DB: strapi
+    volumes:
+        - strapi-data:/var/lib/postgresql/data/ #using a volume
+    #   - ./data:/var/lib/postgresql/data/ # if you want to use a bind folder
+    ports:
+      - '5432:5432'
+    networks:
+      - hmpps
+
+volumes:
+  strapi-data:
 
 networks:
-  hmpps:
+    hmpps:
```

Comment:
Has something happened with the indenting here? We don't have any linting on the dockerfile but should probably keep it clean at 2 space indentation.